css/index_style.css

```diff
@@ -0,0 +1,206 @@
+main section a {
+  width: max-content;
+}
+
+.hero img {
+  width: 100%;
+}
+
+.explore {
+  padding-bottom: 33px;
+}
+
+.discount {
+  background-color: var(--lilac);
+  margin-top: -5px;
+}
+
+.feature-img {
+  width: 70px;
+}
+.feature-img img {
+  width: 100%;
+}
+.vision {
+  background-color: var(--light-Green);
+}
+.vision .flex_col {
+  align-items: center;
+  text-align: center;
+}
+.vision img,
+.wellness_path img,
+.discover img {
+  width: 100%;
+}
+
+.wellness_path {
+  background-color: var(--lilac);
+  padding-left: 30px;
+  padding-right: 30px;
+}
+.wellness_selection {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-areas:
+    "coldpr coldpr coldpr"
+    "smooth smooth smooth"
+    "shot shot shot"
+    "button button button";
+}
+.wellness_coldpr {
+  grid-area: coldpr;
+}
+.wellness_smoothie {
+  grid-area: smooth;
+}
+.wellness_shots {
+  grid-area: shot;
+}
+.wellness_selection a {
+  grid-area: button;
+  justify-self: center;
+}
+.get_juice {
+  background-color: var(--light-Green);
+}
+.circle_border {
+  width: 44px;
+  height: 44px;
+  border: solid 1px var(--black);
+  border-radius: 50%;
+}
+.follow {
+  display: grid;
+  grid-template-rows: auto 1fr 1fr;
+  grid-template-columns: repeat(12, 1fr);
+}
+
+.follow_green {
+  background-color: var(--dark-Green);
+  grid-row: 3 / 4;
+  grid-column: 1 / -1;
+}
+.img_apples {
+  align-self: flex-end;
+  grid-row: 3 / 4;
+  grid-column: 5 / 9;
+}
+.img_papaya {
+  align-self: center;
+  grid-row: 2 / 4;
+  grid-column: 3 / 7;
+}
+
+.img_bag {
+  align-self: flex-start;
+  grid-row: 1 / 3;
+  grid-column: 4 / 8;
+}
+
+.img_kale {
+  align-self: center;
+  grid-row: 2 / 4;
+  grid-column: 7 / 11;
+}
+
+.img_apples img {
+  transform: rotate(333.85071717456117deg);
+  width: 100%;
+}
+.img_papaya img {
+  transform: rotate(16.69666345366653deg);
+  width: 100%;
+}
+.img_bag img {
+  transform: rotate(347.34482520820825deg);
+  width: 100%;
+}
+.img_kale img {
+  transform: rotate(17.142845342601447deg);
+  width: 100%;
+}
+.follow img {
+  width: 100%;
+}
+@media screen and (min-width: 481px) {
```

Comment:
481px is a little too early to be switching out of mobile view. I'd keep mobile view until 600px minimum at this stage. 
Where you decide to 'break' will depend on how many breakpoints you have (i.e. 1 breakpoint = only 2 versions, therefore better to break later. Whereas 4 breakpoints = 5 versions and therefore you'll spread the breakpoints out more).